function-support.html

```diff
@@ -947,6 +931,21 @@ <h2 id="annotation">Annotation</h2>
 		</tr>
 	</table>
 	
+	<div class="indented">
+		<p><code>\tag{hi} x+y^{2x}</code></p>
+		<p>\[\tag{hi} x+y^{2x} \]</p>
+		<p><code>\tag*{hi} x+y^{2x}</code></p>
+		<p>\[\tag*{hi} x+y^{2x} \]</p>
+		<p><code>\tag</code> is not yet supported inside the <code>{gather}</code> environment.</p>
+	</div>	
+	
+<h3 id="line-breaks">Line Breaks</h3>
+	<p>KaTeX 0.10.0+ will insert automatic line breaks in inline math. These can be suppressed by placing math inside a pair of braces, as in <code>{F=ma}</code>.</p>

```

Comment:
Maybe add to the first sentence "after binary operators (like `+` and `\cdot`) and relations (like `=` and `<`) at the top level, like LaTeX".

Reply:
In my first draft, I had this sentence written in almost the exact words that you suggest. I cut it back because I'm trying to avoid explanation that just duplicates LaTeX documentation. When KaTeX behavior differs in subtle ways from LaTeX or MathJax, then I have to explain things, but otherwise this is a document that shows examples but does not explain. That way, I can fit the document onto a single HTML page.

Maybe I'll think of some very concise way to mention where the breaks will come. But if not, I'm inclined to let people learn that by observation or by looking up the LaTeX docs.

Reply:
Maybe just add "like LaTeX" then? I don't want them to think it adds them willy-nilly.  But I'm not sure most people know the rule, so it also wouldn't hurt.  (I certainly didn't!)